client/cmd/dexc-desktop/pkg/sign-windows.cmd

```diff
@@ -0,0 +1,26 @@
+::
+:: This script signs the provided file with the Certificate.pfx file.
+::
+:: Usage:
+:: sign-windows.cmd <file>
+
+if not "%1"=="" (
+    echo Usage: sign-windows.cmd ^<file^>
+    exit /b 1
+)
+
+set scriptDir=%~dp0
+set certFile=%scriptDir%\..\certs\Certificate.pfx
+:: FIXME: obtain the password from command line
+set certPassword=YourPassword
+
+:: TODO: obtain the SDK version from the system
+set sdkVersion=10.0.20348.0
+set signTool="C:\Program Files (x86)\Windows Kits\10\bin\%sdkVersion%\x86\signtool.exe"

```

Comment:
Should figure this out here.

Reply:
Is this actually going to be used?  I didn't want to spend time on this before confirming.  Obtaining a signing cert involves jumping through some hoops, and the existence of some kind of legal entity.

https://support.microsoft.com/en-gb/office/obtain-a-digital-certificate-and-create-a-digital-signature-e3d9d813-3305-4164-a820-2e063d86e512